docs/index.md

```diff
@@ -69,3 +69,44 @@ Maven's `-P` switch: `mvn -Pprm install`.
 Alternatively, to build the tarball execute:
 
     mvn clean install assembly:single
+
```

Comment:
The section is only of interest for developers. Move i to a separated page called `dev.md`. See: https://dans-knaw.github.io/dans-module-archetype/common-practices/#documentation

Also, my first choice would be to connect to the vagrant box used in Data Station development. That said, the Docker approach is still useful as an alternative.